src/Elm/Parser/Tokens.elm

```diff
@@ -163,7 +164,7 @@ functionName =
         { inner =
             \c ->
                 -- checking for these common ranges early is much faster
-                charIsAlphaNumFast c || c == '_' || Unicode.isAlphaNum c
+                Char.Extra.isAlphaNumFast c || c == '_' || Unicode.isAlphaNum c
```

Comment:
Should we try to use a version of `Unicode.isAlphaNum` that doesn't check the same ranges as `Char.Extra.isAlphaNumFast`?